test/fixtures/README.md

```diff
@@ -0,0 +1,23 @@
+Audio file fixtures for the tests.
+
+#### test.wav
+Test.wav was produced by doing:
+
+```py
+import numpy as np
+from scipy.io import wavfile
+
+if __name__ == '__main__':
+    size = 512
+    a = np.full((size, ), 0.)
+    b = np.full((size, ), 0.2)
+    c = np.full((size, ), 0.5)
+    d = np.full((size, ), 0.9)
+    t = np.concatenate((a, b, c, d))
+
+    wavfile.write('test.wav', 44100, t)
+```
+
+#### wavetest.wav
+
+Produced with `make_test_wave.py`
```

Comment:
It's a little confusing that there are files called `test.wav` and `wavtest.wav`. Maybe it would be useful to describe what each one is for, and how they're different?

Reply:
I'm using `test.wav` for the `macca` backend--- I'll find some better names for it. We also might not need both